cli/src/commands/workspace.rs

```diff
@@ -61,10 +61,22 @@ pub(crate) struct WorkspaceAddArgs {
     /// directory.
     #[arg(long)]
     name: Option<String>,
-    /// The revision that the workspace should be created at; a new working copy
-    /// commit will be created on top of it.
+    /// A list of parent revisions for the new workspace and its working-copy
+    /// commit. May be empty, a single revision, or multiple revisions.
+    ///
+    /// If no revision is provided (the default), the new workspace will be
+    /// created, and its working-copy commit will exist on top of the parent(s)
+    /// of the working-copy commit in the current workspace, i.e. they will
+    /// share the same parent.
+    ///
+    /// If a single revision is provided, the new working-copy commit will be
+    /// created on top of that revision.
+    ///
+    /// If multiple commits are provided, the new working-copy commit will be
+    /// created on top of all of them, i.e. they will be merged as if you had
+    /// used `jj new r1 r2 r3 ...`.
```

Comment:
I don't really like that this makes it sound like we treat a single parent differently from multiple parents, but maybe it's still best to describe it this way. I. might write it like this:

> If any revisions are provided, the new working-copy commit will be created on top of them. So, if
> more than one revisions are provided, the new working-copy commit will be a merge commit.

But feel free to keep the current version if you think it's clearer. I don't really know.

Reply:
I gave it another rewording, let me know what you think.